Source/WebCore/svg/SVGLengthValue.cpp

```diff
@@ -271,6 +271,31 @@ SVGLengthValue SVGLengthValue::blend(const SVGLengthValue& from, const SVGLength
     return { WebCore::blend(fromValue.releaseReturnValue(), toValue, { progress }), to.lengthType() };
 }
 
+WebCore::LengthType SVGLengthValue::toCoreLengthType() const {
```

Comment:
the manual effort of mapping SVGLengthType to LengthType feels like it's taking the wrong direction. there should be a more standardized type that supports interoperability between these.